limitador-server/sandbox/docker-compose-limitador-distributed.yaml

```diff
@@ -0,0 +1,28 @@
+---
+version: '3.8'
+services:
+  limitador:
+    image: limitador-testing-all-features
+    build:
+      context: ../..
+      dockerfile: Dockerfile
+      args:
+        - CARGO_ARGS=--all-features
+    depends_on:
+      - envoy
```

Comment:
envoy is not being used, is it?

Reply:
All the other limitador compose files have this dependency.  But Yeah that seems backwards.

Reply:
yeah, exactly.

We are pushing the current "unique" sandbox too far. In other PR, we could refactor to have different sandboxes, each one running only necessary services. Each sandbox with custom readme.md and docker-compose.yaml file.

Reply:
I fixed the dependency on this compose file and the other ones too.